lib/chef/provisioning/docker_driver/version.rb

```diff
@@ -1,7 +1,7 @@
 class Chef
 module Provisioning
 module DockerDriver
-  VERSION = '0.5.2'
+  VERSION = '0.7'
```

Comment:
If you're going to do this in this PR I would also like to see CHANGELOG updates before releasing (and it should be 0.7.0)